packages/framework/client-logger/app-insights-logger/src/fluidAppInsightsLogger.ts

```diff
@@ -9,6 +9,56 @@ import {
 	type ITelemetryBaseLogger,
 } from "@fluidframework/core-interfaces";
 
+/**
+ * An Enum of Fluid telemetry event categories
+ */
+export const TelemetryEventCategory = {
+	PERFORMANCE: "performance",
+	GENERIC: "generic",
+	ERROR: "error",
+} as const;
+
+/**
+ * The type for the enum of Fluid telemetry event categories
+ */
+export type TelemetryEventCategory =
+	(typeof TelemetryEventCategory)[keyof typeof TelemetryEventCategory];
+
+/**
+ * The configuration object for the {@link FluidAppInsightsLogger}
+ */
+export interface FluidAppInsightsLoggerConfig {
+	filterConfig: {
+		/**
+		 * Determines whether all telemetry events are sent or not sent by default and whether filters will exclude matching telemetry events or include them.
+		 *
+		 * "inclusive" mode means all logs are NOT SENT by default and only the events that match specified filters will be sent (included).
+		 *
+		 * "exclusive" mode means all logs ARE SENT by default and only the events that match specified filters will be  not be sent (excluded).
+		 */
+		mode: "inclusive" | "exclusive";
+		/**
+		 * Controls the default filtering of log events by their category.
+		 * This can be overriden with namespace level filters
+		 */
+		filters?: TelemetryFilter[];
```

Comment:
What does it mean if this is `undefined` and mode is set to `inclusive`? Is it equivalent to an empty array?

Reply:
Probably worth calling that out explicitly in the docs, since the case is slightly ambiguous.